pkg/jx/cmd/create_vault.go

```diff
@@ -364,3 +370,7 @@ func (o *CreateVaultOptions) exposeVault(vaultService string) error {
 	options.Services = []string{vaultService}
 	return options.Run()
 }
+
+func (o *CreateVaultOptions) clusterName() (string, error) {
+	return gke.ShortClusterName(o.Kube())
```

Comment:
should this be tied to GKE here? others that use GKE have GKE in the function name so it is obvious